src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -2159,28 +2210,51 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="ConfirmButton" type="MarginContainer" parent="MicrobeEditorGUI/CellEditor"]
+[node name="BottomRight" type="MarginContainer" parent="MicrobeEditorGUI/CellEditor"]
 anchor_left = 1.0
 anchor_top = 1.0
 anchor_right = 1.0
 anchor_bottom = 1.0
-margin_left = -150.0
-margin_top = -50.0
+margin_right = -10.0
+margin_bottom = -10.0
 grow_horizontal = 0
-rect_min_size = Vector2( 150, 0 )
-custom_constants/margin_right = 10
-custom_constants/margin_bottom = 10
+grow_vertical = 0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Button" type="Button" parent="MicrobeEditorGUI/CellEditor/ConfirmButton"]
+[node name="HBoxContainer" type="HBoxContainer" parent="MicrobeEditorGUI/CellEditor/BottomRight"]
 margin_right = 172.0
 margin_bottom = 40.0
+alignment = 2
+
+[node name="CancelButton" type="Button" parent="MicrobeEditorGUI/CellEditor/BottomRight/HBoxContainer"]
+visible = false
+margin_right = 223.0
+margin_bottom = 40.0
+rect_min_size = Vector2( 140, 40 )
 focus_mode = 0
 mouse_filter = 1
 custom_styles/hover = SubResource( 23 )
-custom_styles/normal = SubResource( 24 )
+custom_styles/pressed = SubResource( 24 )
+custom_styles/normal = SubResource( 25 )
+custom_colors/font_color_disabled = Color( 0.498039, 0.529412, 0.541176, 1 )
+custom_colors/font_color_pressed = Color( 1, 1, 1, 1 )
+shortcut_in_tooltip = false
+enabled_focus_mode = 0
+text = "CANCEL_MOVE_CAPITAL"
```

Comment:
As I said earlier I think the button should be named "Cancel Action" instead of being specific to moving, that way it is more reusable in the future.